defaults/main.yml

```diff
@@ -30,3 +30,9 @@ dr_clean_orphaned_disks: "True"
 
 # Indicate the default entities status report file name
 dr_report_file: "report.log"
+
+# Indicate the file name which is used to contain the data of the running VMs in the secondary setup before the failback
+# to be run again in the primary setup after the failback will be finished.
+dr_json_running_vms: "json_running_vms"
```

Comment:
It's not used anywhere

Reply:
What is not used? the main.yml?

Reply:
Right, c/p mistake, I used dr_report_file instead of dr_json_running_vms.
Thanks for the update